front/src/Phaser/Game/GameScene.ts

```diff
@@ -830,48 +835,77 @@ ${escapedMessage}
                 id++;
             }
             this.tweens.add({
-                targets     : domElement ,
-                scale       : 1,
-                ease        : "EaseOut",
-                duration    : 400,
+                targets: domElement,
+                scale: 1,
+                ease: "EaseOut",
+                duration: 400,
             });
 
             this.popUpElements.set(openPopupEvent.popupId, domElement);
         }));
 
-       this.iframeSubscriptionList.push(iframeListener.closePopupStream.subscribe((closePopupEvent) => {
+        this.iframeSubscriptionList.push(iframeListener.closePopupStream.subscribe((closePopupEvent) => {
             const popUpElement = this.popUpElements.get(closePopupEvent.popupId);
             if (popUpElement === undefined) {
-                console.error('Could not close popup with ID ', closePopupEvent.popupId,'. Maybe it has already been closed?');
+                console.error('Could not close popup with ID ', closePopupEvent.popupId, '. Maybe it has already been closed?');
             }
 
             this.tweens.add({
-                targets     : popUpElement ,
-                scale       : 0,
-                ease        : "EaseOut",
-                duration    : 400,
-                onComplete  : () => {
+                targets: popUpElement,
+                scale: 0,
+                ease: "EaseOut",
+                duration: 400,
+                onComplete: () => {
                     popUpElement?.destroy();
                     this.popUpElements.delete(closePopupEvent.popupId);
                 },
             });
         }));
 
-       this.iframeSubscriptionList.push(iframeListener.disablePlayerControlStream.subscribe(()=>{
+        this.iframeSubscriptionList.push(iframeListener.disablePlayerControlStream.subscribe(() => {
             this.userInputManager.disableControls();
         }));
 
-       this.iframeSubscriptionList.push(iframeListener.enablePlayerControlStream.subscribe(()=>{
+        this.iframeSubscriptionList.push(iframeListener.enablePlayerControlStream.subscribe(() => {
             this.userInputManager.restoreControls();
         }));
-        let scriptedBubbleSprite : Sprite;
-       this.iframeSubscriptionList.push(iframeListener.displayBubbleStream.subscribe(()=>{
-            scriptedBubbleSprite = new Sprite(this,this.CurrentPlayer.x + 25,this.CurrentPlayer.y,'circleSprite-white');
+        this.iframeSubscriptionList.push(iframeListener.gameStateStream.subscribe(() => {
+            /*const playerObject: PlayerStateObject = {
+                [this.playerName]: {
+                    position: {
+                        x: this.CurrentPlayer.x,
+                        y: this.CurrentPlayer.y
+                    },
+                    pusherId: this.connection?.getUserId()
+                }
+            }
+            for (const mapPlayer of this.MapPlayers.children.entries) {
```

Comment:
https://github.com/thecodingmachine/workadventure/issues/1087

maybe we should readd the informatin after all 🤔 though i dont think handling the remote players as a callback on update would be a good idea as it would probably slow down the application too much